chef/cookbooks/hyperv/recipes/windows_features.rb

```diff
@@ -15,6 +15,7 @@
   node[:features_list][:windows].each do |feature_list|
     windows_feature feature_list do
       action :install
+      all true
```

Comment:
@vuntz that would be a good idea, because this is the best way to solve this. We need the "all" attribute to automatically install the dependencies